dalc/celestia-light/config.toml

```diff
@@ -2,7 +2,7 @@
   Remote = false
   [Core.RemoteConfig]
     Protocol = "tcp"
-    RemoteAddr = "192.167.10.2:26657"
+    RemoteAddr = "192.167.10.0:26657"
```

Comment:
Why this change?

Reply:
This is the correct value, but because we currently spin up 3 core nodes that serve on 192.167.10.0 , 192.167.10.1, and 192.167.10.2 it worked before.